vrk-kpa/suomidigi
# SUOPA-905 Bullet points

To test: 
- `make fresh`
- Log in and go to edit f..e https://suomidigi.docker.sh/node/779/edit
- Add new paragraph Highligh bullet point (Korostettu lista)
   - Add bullet points and select a bullet point style for each bullet
- Save the node and check that the layout for the bullet point list is similar to https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/769eb9fe-63b7-4df0-9c3e-732244b05fc5/Suomidigi-2020-artikkeli-xl/
